src/__tests__/__fixtures__/explicit-ssr/output.js

```diff
@@ -0,0 +1,16 @@
+import { lazyForPaint } from 'react-loosely-lazy';
+const ExplicitSsr = lazyForPaint(
+  () => {
+    const resolved = require('./my-component');
```

Comment:
Can we have two sets of test suites / plugin runners that run the plugin in client and server mode? Otherwise we do not validate all cases correctly, and the test output is quite confusing with implicit client values. Also, I think we should add all combinations of `lazy`, `lazyForPaint`, and `lazyAfterPaint` which can be in each fixture.

Reply:
Yep cool will do tests for both client and server for all use cases